### PR TITLE
GetAddr function added to Config

### PR DESCRIPTION
**Pull request by blkmlk**
The idea came from the discussion (https://github.com/centrifugal/gocent/pull/11)

### CODE REVIEW

File: client.go [hunk 1]
```diff
@@ -58,20 +63,17 @@ var DefaultHTTPClient = &http.Client{Transport: &http.Transport{
 
 // New returns initialized client instance based on provided config.
 func New(c Config) *Client {
-	addr := strings.TrimRight(c.Addr, "/")
-	if !strings.HasSuffix(addr, "/api") {
```
Comment (FZambia) [inline client.go:62]: We can't remove this without new major version release - I think it should stay the same till v3

Comment (blkmlk) [reply client.go]: Returned back

**Comment by FZambia** [comment]
@blkmlk thanks! Released v2.2.0 with this functionality.

**Comment by blkmlk** [comment]
Thank you for merging!

**Pull request merged**